samtools.wdl

```diff
@@ -177,17 +178,25 @@ task Fastq {
 
 task Tabix {
     input {
-        String inputFile
+        File inputFile
+        String outputFilePath = "indexed.vcf.gz"
```

Comment:
Again, should this have a default?

Reply:
Yes. These absolute paths things we use are only relevant for shared filesystems. For cloud you just point to the outputs. So then it does not matter how these are named.